jwql/instrument_monitors/nirspec_monitors/ta_monitors/wata_monitor.py

```diff
@@ -677,7 +677,7 @@ def most_recent_search(self):
             where the wata monitor was run.
         """
         query = session.query(self.query_table).filter(and_(self.query_table.aperture == self.aperture,
-                                                            self.query_table.run_monitor == True)).order_by(self.query_table.end_time_mjd).all()
+                                                            self.query_table.run_monitor == True)).order_by(self.query_table.end_time_mjd).all() # noqa: E712
```

Comment:
another truth check, do we need it?

Reply:
Same here as above. I believe we need to keep this